tensorflow/compiler/xla/service/cpu/parallel_loop_emitter.h

```diff
@@ -61,7 +61,8 @@ class ParallelLoopEmitter : public llvm_ir::LoopEmitter {
   ~ParallelLoopEmitter() override = default;
 
   std::vector<llvm_ir::IrArray::Index> EmitIndexAndSetExitBasicBlock(
-      absl::string_view loop_name, llvm::Type* index_type) override;
+      absl::string_view loop_name, llvm::Type* index_type,
+      llvm::Value* base_index = nullptr) override;
```

Comment:
Can you remove the defaulted `nullptr` from virtual functions? See https://google.github.io/styleguide/cppguide.html#Default_Arguments